altair/vegalite/v5/schema/vega-themes.json

```diff
@@ -28,12 +28,6 @@
     "circle": {
       "fill": "#6929c4"
     },
-    "group": {
-      "fill": "#ffffff"
-    },
-    "path": {
-      "stroke": "#6929c4"
-    },
```

Comment:
See 79804ec6b1ee009c83d6993e390a040832e3bafc

Reply:
Wow nice! Wasn't aware that such cases exist